org.scala-ide.sdt.debug/plugin.xml

```diff
@@ -142,17 +142,20 @@
          point="org.eclipse.ui.menus">
       <menuContribution
             allPopups="false"
-            locationURI="toolbar:org.scala-ide.sdt.debug.asyncView">
-         <command
-               commandId="org.scala-ide.sdt.debug.stepMessageOut"
-               icon="icons/bang.png"
-               label="Step Message Out"
-               style="push"
-               tooltip="Step until a message is received">
-            <visibleWhen
-                  checkEnabled="true">
-            </visibleWhen>
-         </command>
+            locationURI="toolbar:org.eclipse.ui.main.toolbar">
```

Comment:
looks this is better but needs to be verified:
            locationURI="toolbar:org.eclipse.debug.ui.DebugView"